public/app/features/alerting/EditNotificationChannelPage.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { PureComponent } from 'react';
+import { MapDispatchToProps, MapStateToProps } from 'react-redux';
+import { NavModel } from '@grafana/data';
+import { config } from '@grafana/runtime';
+import { Form } from '@grafana/ui';
+import Page from 'app/core/components/Page/Page';
+import { NotificationChannelForm } from './components/NotificationChannelForm';
+import {
+  loadNotificationChannel,
+  loadNotificationTypes,
+  testNotificationChannel,
+  updateNotificationChannel,
+} from './state/actions';
+import { getNavModel } from 'app/core/selectors/navModel';
+import { getRouteParamsId } from 'app/core/selectors/location';
+import { selectableChannels, transformSubmitData, transformTestData } from './utils/notificationChannels';
+import { NotificationChannelType, NotificationChannelDTO, StoreState } from 'app/types';
+import { connectWithCleanUp } from '../../core/components/connectWithCleanUp';
+import LoadingIndicator from '@jaegertracing/jaeger-ui-components/src/common/LoadingIndicator';
```

Comment:
What is our approach to using Jaeger components?

Reply:
think this is just a mistaken import ,we have our own  LoadingIndicator

Reply:
Bah